tms_objects2.erb

```diff
@@ -432,6 +439,7 @@ objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery]
 #7 - eventMaterialsTech - conceptID/term - Has all(colThesxrefType,UCTS,FinalST,term) 
 #6113 - ExhTextEntries |t| - eventID - TextType - LUX exhibition
 #4999 - ExhTextEntries |t| - eventID - TextType - ExhibitionURL
+#586 - ExhVenuesAltName |an| - appellationValue - AltVenueName - NULL
```

Comment:
I think this query would give the objects where venue doesn't have a name but it returns nothing:

  select distinct a.ObjectID, b.ObjectID
  from Coboat_ExhVenues a
  left join Coboat_ExhVenuesAltName b on a.ExhVenueXrefID = b.ExhVenueXrefID
  where b.objectID is null
  order by a.objectID 

so put in comment null doesn't happen for exhibition nameactor set appellationvalue